### PR TITLE
Add debian installation instructions to README

### PR DESCRIPTION
**Pull request by alexDickhans**

### CODE REVIEW

**Review by lewisfm — CHANGES_REQUESTED** [review]
Can you move the duplicated pip/pros-cli install lines into their own section? I noticed you did that for the "apt install pip" one but maybe forget to remove it from the first two sections.

File: README.md [hunk 1]
```diff
@@ -72,6 +72,37 @@ rustup component add rust-src
...
+
+Install pip:
+```console
+sudo apt install python3-pip
+```
+
+Close and reopen the terminal, and install pros-rs:
+
+```console
+rustup component add rust-src --toolchain nightly-x86_64-unknown-linux-gnu
```
Comment (lewisfm) [inline README.md]: Also I think this can just be "nightly" since I'm not sure its a good idea to assume everyone's running x64.

Comment (alexDickhans) [reply README.md]: Yep - just fixed it

**Review by lewisfm — APPROVED** [review]
:+1:

**Pull request merged**